### DISCUSSION TITLE
Can Compliance Scripts be run independently?

### DISCUSSION THREAD

**Discussion by trbryant97** [usnistgov/macos_security#441] [Q&A] [answered]
I am in the process of updating MacOS STIG scripts for my project, after generating CMMC 2.0 compliance I modified the script to work with my current MDM tool. As I am testing the shell script, it seems to fail almost immediately when run independently. Are the compliance shell scripts dependent on any other files?

**Comment by jmahlman** [comment]
How are you running the script? Can you include the commands and output? The compliance scripts can be run interactively and non-interactively with flags.

**Reply by trbryant97** [reply, marked as answer]
Hello @jmahlman since posting this discussion I have discovered that my script is failing due to permissions and must be run as sudo/root.
